index.html

```diff
@@ -0,0 +1,22 @@
+<html>
+
+<head>
+  <script src="script.js"></script>
+  <style>
+    body {
+      background-color: skyblue;
+    }
```

Comment:
If you've only got a single style like this it's cool to inline it here. In that case you'd delete the `.css` file. Once you've got a bunch of styles then you'd want to move them to the file rather than inlining since it will start to get messy.